kolibri/core/webpack/hooks.py

```diff
@@ -230,7 +230,7 @@ def frontend_message_file(self, lang_code):
             file_path = os.path.join(
                 path,
                 to_locale(lang_code),
-                "LC_FRONTEND_MESSAGES",
+                "LC_MESSAGES",
```

Comment:
Is this just to get everything in the same folder on crowdin?

Reply:
no, on crowdin we skip this folder entirely.

I would have preferred to mirror the crowdin directory structure in our repo, but it seems that the assumed existence of an `LC_MESSAGES` directory is built into the Django translation machinery.

As far as I can tell, adding a separate `LC_FRONTEND_MESSAGES` was never actually necessary and may have just been an off-the-cuff developer decision